src/editor/EditorStatusBar.js

```diff
@@ -226,6 +227,25 @@ define(function (require, exports, module) {
     }
 
     /**
+     * Update LineEndings label
+     * @param {Event} event (unused)
+     * @param {Editor} editor Current editor
+     * @param {boolean=} doNotAnimate True if state should not be animated
+     */
+    function _updateLineEndingsLabel(event, editor, doNotAnimate) {
+        /* Toggle text */
+        if($statusLineEndings.text() === Strings.STATUSBAR_LINE_ENDINGS_CRLF) {
+            $statusLineEndings.text(Strings.STATUSBAR_LINE_ENDINGS_LF);
+        } else {
+            $statusLineEndings.text(Strings.STATUSBAR_LINE_ENDINGS_CRLF);
+        }
+        
+        if (!doNotAnimate) {
+            AnimationUtils.animateUsingClass($statusLineEndings[0], "flash", 1500);
+        }
```

Comment:
This code is never used.  The animation effect is pretty specific to the insert/overwrite toggle, so I don't think we'll ever need this here.